codalab/requirements/common.txt

```diff
@@ -57,7 +57,7 @@ gunicorn==19.7.0
 #gevent==1.2.2
 
 #Extras added
-git+https://github.com/ikraft/django-userswitch.git
+git+https://github.com/Tthomas63/django-switchuser.git
```

Comment:
can you put in a PR to the original repo with your fix please? And if they don't accept we'll go this route

Reply:
https://github.com/wolever/django-switchuser/pull/8